hack/testing/rsyslog/roles/openshift_logging_rsyslog/tasks/main.yaml

```diff
@@ -0,0 +1,122 @@
+---
+- name: see if openshift-logging namespace exists
```

Comment:
These tasks through line 79 should only be run on the masters[0] node, and are specific to openshift.  The remaining tasks are not specific to openshift and should be run on all nodes.

Reply:
Do you mean until line 70?

Reply:
No, I mean these tasks - the `oc` command is only guaranteed to be present on the master nodes with the typical openshift installation

Reply:
There is no line 79...
Only 70 lines in the file.

Reply:
hmm - ok - then - Any task which uses `oc` must be executed on node `masters[0]`

Reply:
I run the openshift related tasks only on master[0] here in the playbook (at least, I meant... :)
https://github.com/nhosoi/origin-aggregated-logging/blob/rsyslog_deploy/hack/testing/rsyslog/playbook.yaml#L11-L16

Reply:
I'm not familier with this syntax in the playbook.
When I usually want to run once on a specific machine I either have a new separate play for the specific hosts 
or delegate the tasks to the machine and add run_ once: true flag to the tasks.

Does the syntax you are using work for you?

Reply:
Well, it was not working as I expected... :(
The current source tree passes the rsyslog ci-test.  So, I did not notice this warning was issued from ansible.
``` [WARNING]: Ignoring invalid attribute: hosts```
Obviously, ```hosts: master[0]``` was ignored.  I'm going to follow your suggestion.  
Thanks, @sradco !

Reply:
Where did you use hosts? I would recommend keeping with the openshift way of doing things for the openshift specific playbooks.

Reply:
I think these 2 lines would do the trick.  Could you please review it?
https://github.com/nhosoi/origin-aggregated-logging/blob/rsyslog_deploy/hack/testing/rsyslog/playbook.yaml#L15-L16

Reply:
something like this: https://gist.github.com/richm/f76f723ebd1ab1ec8d34f5b225abed20

That is, before doing the main rsyslog playbook, do the openshift specific bits to get the certs, tokens, etc.

In the future, we'll have `rsyslog_` or `logging_` ansible variables for certs, tokens, etc. e.g. `logging_mmk8s_token`, `logging_mmk8s_cacert`, etc. that we will specify in an inventory or vars.yaml before passing to the rsyslog playbook.

Reply:
Thanks, @richm .  Verified your playbook.yaml and pushed it.

Reply:
And added these variables...
+logging_mmk8s_token: "{{rsyslog__viaq_path}}/mmk8s.token"
+logging_mmk8s_ca_cert: "{{rsyslog__viaq_path}}/mmk8s.ca.crt"
+logging_elasticsearch_ca_cert: "{{rsyslog__viaq_path}}/es-ca.crt"
+logging_elasticsearch_cert: "{{rsyslog__viaq_path}}/es-cert.pem"
+logging_elasticsearch_key: "{{rsyslog__viaq_path}}/es-key.pem"